demos/ble/mqtt_ble/mqtt_demo_ble.cmake

```diff
@@ -0,0 +1,30 @@
+# MQTT over BLE demo
+afr_demo_module(mqtt_ble)
+
+afr_set_demo_metadata(ID "MQTT_BLE_TRANSPORT_DEMO")
+afr_set_demo_metadata(DESCRIPTION "An example that demonstrates MQTT over BLE assisted by a companion device")
+afr_set_demo_metadata(DISPLAY_NAME "MQTT Hello World demo over BLE")
+
+afr_module_sources(
+    ${AFR_CURRENT_MODULE}
+    INTERFACE
+        "${CMAKE_CURRENT_LIST_DIR}/mqtt_demo_ble_transport.c"
```

Comment:
The source list should include the `mqtt_demo_ble.cmake` so that the metadata contains the file.
Here is the MQTT library configuration for reference: https://github.com/aws/amazon-freertos/blob/master/libraries/core_mqtt_demo_dependencies.cmake#L23-L25

Reply:
Added.